### ISSUE TITLE
SSH to containers fails intermittently

### ISSUE THREAD

**Issue by hughsaunders** [rcbops/ansible-lxc-rpc#358]
For example...

```
01:34:49.490 TASK: [utility_create_configs | Create all role checks for each host] ********* 
01:34:50.183 fatal: [node20_cinder_api_container-6e53ce47] => SSH encountered an unknown error during the connection. We recommend you re-run the command using -vvvv, which will enable SSH debugging output to help diagnose the issue
01:34:50.190 fatal: [node18_nova_conductor_container-f0c78ea3] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
01:34:50.191 fatal: [node18_nova_cert_container-bc9363a8] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
```

The containers that have the error have been contacted successfully multiple times previously during the run.

**Comment by hughsaunders** [comment]
Another variant of the SSH error: 
http://jenkins.propter.net/job/dev-sat6-gate/201/consoleFull

```
11:06:48 TASK: [rsyslog_config | Deploy rsyslog config generator] ********************** 
11:06:48 fatal: [node25_rsyslog_container-ba91fd29] => SSH encountered an unknown error during the connection. We recommend you re-run the command using -vvvv, which will enable SSH debugging output to help diagnose the issue
```

**Comment by hughsaunders** [comment]
http://jenkins.propter.net/job/dev-sat6-gate/212/consoleFull

**Comment by hughsaunders** [comment]
http://jenkins.propter.net/job/dev-sat6-gate/218/consoleFull

**Comment by hughsaunders** [comment]
http://jenkins.propter.net/job/dev-sat6-gate/221/consoleFull

```
00:09:28.630 GATHERING FACTS *************************************************************** 
00:09:29.125 fatal: [node29_galera_container-3619c9d1] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:29.853 fatal: [node29_glance_container-35624074] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:30.537 fatal: [node29_horizon_container-d583b303] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.901 fatal: [node29_utility_container-c0591f64] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.909 fatal: [node29_nova_scheduler_container-4f8e5d81] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.910 fatal: [node29_nova_conductor_container-1d025660] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.921 fatal: [node29_cinder_api_container-caf5e8db] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.922 fatal: [node29_keystone_container-a4aad3be] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.945 fatal: [node29_nova_cert_container-8de57ce9] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.949 fatal: [node29_rabbit_mq_container-0d96ae4c] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.957 fatal: [node29_nova_api_ec2_container-3a1e88c3] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.961 fatal: [node29_neutron_server_container-53c6bbb9] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:31.973 fatal: [node29_nova_spice_console_container-a52d9b9e] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:32.007 ok: [node37_rsyslog_container-af374338]
00:09:32.093 ok: [node38_rsyslog_container-f4b4df6e]
00:09:32.365 ok: [node38_cinder_volumes_container-ddacb0ec]
00:09:32.990 fatal: [node29_neutron_agents_container-07a6faff] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:33.281 fatal: [node28_horizon_container-297323ed] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:33.681 fatal: [node29_nova_api_os_compute_container-7df0a865] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:33.954 fatal: [node28_nova_api_ec2_container-e6ea4c6e] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:34.750 fatal: [node28_keystone_container-0ef84528] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:34.930 fatal: [node29_nova_api_metadata_container-ca3eab65] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.021 fatal: [node29_rsyslog_container-d2825dc7] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.045 fatal: [node29_memcached_container-b0961f0c] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.053 fatal: [node29_heat_engine_container-4b0a5475] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.061 fatal: [node29_heat_apis_container-0c9632fe] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.077 fatal: [node28_elasticsearch_container-37cff10a] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.097 fatal: [node28_nova_scheduler_container-ff75bff6] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.101 fatal: [node28_cinder_api_container-429c8548] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.117 fatal: [node28_nova_conductor_container-be2ebc27] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.137 fatal: [node28_utility_container-a079c9c9] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.422 fatal: [node28_nova_api_os_compute_container-f68c9bec] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:35.509 fatal: [node28_nova_cert_container-c20df8d9] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:36.105 fatal: [node28_galera_container-37fe8c3f] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:36.166 fatal: [node28_nova_spice_console_container-c78c4f3f] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:36.221 fatal: [node28_kibana_container-365f67f5] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:37.070 fatal: [node28_rabbit_mq_container-806a7eb0] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:37.686 fatal: [node28_nova_api_metadata_container-26dd4b87] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:37.886 fatal: [node28_glance_container-90b7022b] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.146 fatal: [node28_rsyslog_container-f71a85e3] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.186 fatal: [node28_heat_engine_container-8774f309] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.197 fatal: [node28_logstash_container-4a742de6] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.229 fatal: [node28_heat_apis_container-2bea5f69] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.230 fatal: [node28_memcached_container-f29f5bab] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.257 fatal: [node36_nova_scheduler_container-f939150b] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.281 fatal: [node36_nova_conductor_container-2933ffd6] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.554 fatal: [node36_cinder_api_container-679e1e77] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.634 fatal: [node36_horizon_container-319ab504] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:38.918 fatal: [node36_nova_api_os_compute_container-c911d53a] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:39.222 fatal: [node36_utility_container-8a1af7ea] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:39.290 fatal: [node36_keystone_container-6d0b9a05] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:39.362 fatal: [node36_nova_cert_container-9f86e9fd] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:39.646 fatal: [node36_nova_spice_console_container-a89ef0b2] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:40.186 fatal: [node36_galera_container-c24c12b0] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:40.826 fatal: [node36_nova_api_ec2_container-d0c845cf] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:41.262 fatal: [node36_rabbit_mq_container-b6c3fb40] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:41.318 fatal: [node36_glance_container-e5d7b867] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:41.350 fatal: [node36_rsyslog_container-86ab920a] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:41.362 fatal: [node36_heat_engine_container-6812d316] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:41.406 fatal: [node36_nova_api_metadata_container-4ca7ca1d] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:41.425 fatal: [node36_heat_apis_container-c7a5dc64] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
00:09:41.666 fatal: [node36_memcached_container-0f9dbacb] => SSH Error: data could not be sent to the remote host. Make sure this host can be reached over ssh
```

**Comment by hughsaunders** [comment]
http://jenkins.propter.net/job/dev-sat6-gate/228/consoleFull

**Comment by sigmavirus24** [comment]
Fixed in #359, backported in #447 and #448
